### PR TITLE
Add terraform-docs version

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds a [`terraform-docs`](https://github.com/terraform-docs/terraform-docs) version to `entrypoint.sh`.

## 💭 Motivation and context ##

See [this pull request comment](https://github.com/cisagov/skeleton-generic/pull/82#discussion_r667219358).

## 🧪 Testing ##

All `pre-commit` hooks pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
 
-echo "PACKER_VERSION=1.6.6" >> "$GITHUB_ENV"
-echo "TERRAFORM_VERSION=0.12.31" >> "$GITHUB_ENV"
+{ echo "PACKER_VERSION=1.6.6"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=0.14.1"; } >> "$GITHUB_ENV"
```
Comment (mcdonnnj) [inline entrypoint.sh]: The [versioning for `terraform-docs`](https://github.com/terraform-docs/terraform-docs/releases) has a leading `v`.

```suggestion
{ echo "PACKER_VERSION=1.6.6"; echo "TERRAFORM_VERSION=0.12.31"; echo "TERRAFORM_DOCS_VERSION=v0.14.1"; } >> "$GITHUB_ENV"
```

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
Thanks for taking care of this side! Some great improvement and it's good to get this under version control for consistency. Just one fix and I should be good to approve this PR.

File: entrypoint.sh (hunk 1 above)
Comment (jsf9k) [reply entrypoint.sh]: Great catch!

**Review by mcdonnnj — APPROVED** [review]
Excellent work! I threw this into a workflow in my personal GHA testing repo and it looks like everything installs as expected:

```console
packer --version
terraform --version
terraform-docs --version
shell: /usr/bin/bash -e {0}
env:
  CURL_CACHE_DIR: ~/.cache/curl
  PIP_CACHE_DIR: ~/.cache/pip
  PRE_COMMIT_CACHE_DIR: ~/.cache/pre-commit
  RUN_TMATE: 
  PACKER_VERSION: 1.6.6
  TERRAFORM_VERSION: 0.12.31
  TERRAFORM_DOCS_VERSION: v0.14.1
  pythonLocation: /opt/hostedtoolcache/Python/3.9.5/x64
  LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.9.5/x64/lib
  GOROOT: /opt/hostedtoolcache/go/1.16.5/x64
  GO_VERSION: 1.16.5
1.6.6
Terraform v0.12.31

Your version of Terraform is out of date! The latest version
is 1.0.2. You can update by downloading from https://www.terraform.io/downloads.html
terraform-docs version v0.14.1 linux/amd64
```

Nice shades Frasier!

**Comment by jsf9k** [comment]
> Nice shades Frasier!

Thanks for testing it. :bowing_man:

**Review by dav3r — APPROVED** [review]
Yessir, I like this! 👍

**Pull request merged**